### ISSUE TITLE
Using docker-compose

### ISSUE THREAD

**Issue by sebplorenz** [International-Data-Spaces-Association/IDS-testbed#57]
Hi,
as discussed earlier, I would suggest to use docker-compose for setting up the test bed. With docker compose we:
1. can reduce the code in the install script
2. can use the containers that are already provided by the component repositories (in addition, test result would refer to the provided containers, not self-build containers)
4. do not need an uninstall script
5. can reduce startup time (no container building)

Is anyone already working on this? @aitorce, @jfernandezsqs? If not, I could provide an initial PR.

**Comment by aitorce** [comment]
Hi,

The docker-compose for the Dataspace Connector is working and will soon be uploaded.

We are currently working with the Omejdn DAPS' provided resources.

**Comment by sebplorenz** [comment]
If you are facing issues let me know. I managed to set up the DAPS locally for testing. If you put your current development on a branch here, then I could help.

**Comment by aitorce** [comment]
Hi @sebplorenz,

I have pushed a new branch `integrating-daps-1.6.0` which involves the `docker-compose` for the Dataspace Connector v7.0.2 and I have added our current (`currentDAPS`) https Omejdn DAPS with locally built nginx and daps images and the newer (`newDAPS`) version of the Omejdn DAPS v1.6.0.

I have tried following the new documentation as it seems that both of the the Omejdn DAPS have overlapping directories, such as `config` and `keys`. Let me know if you manage to find the issue in `newDAPS`.

The TLS for nginx is in `/certs` and what was previously called the `signing_key` is in `/omejdn-server/keys/omejdn`.
